IPython/core/display.py

```diff
@@ -467,16 +467,26 @@ def _repr_markdown_(self):
 class Math(TextDisplayObject):
 
     def _repr_latex_(self):
-        s = self.data.strip('$')
-        return "$$%s$$" % s
+        if self.format == u'latex' or not self.format:
+            s = self.data.strip('$')
+            return "$$%s$$" % s
+        
+    def _repr_html_(self):
+        if self.format == u'mathml':
+            return self.data
```

Comment:
The intention here is to render LaTeX by default, but if a format switch is used `Math("<math>...</math>", format="mathml")` is used, then render as HTML.